### PR TITLE
Feature/ilusha

### PR DESCRIPTION
**Pull request by joker222222**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[*.{js,jsx,mjs,cjs,ts,tsx,mts,cts,vue}]
+charset = utf-8
+indent_size = 4
+indent_style = space
```
Comment (ontalex) [inline .editorconfig:4]: why "indent_size"

File: src/assets/fonts.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  font-weight: 200;
+  font-style: italic;
+  font-display: block;
+  src: url('/fonts/SEProDisplay/SF-Pro-Display-ThinItalic.otf') format('opentype');
+}
+
+/* Variables */
+
+:root {
+  --font-family-display: 'SF Pro Display';
```
Comment (ontalex) [inline src/assets/fonts.css:88]: good

**Review by ontalex — APPROVED** [review]
...

File: .editorconfig (hunk 1 above)
Comment (alkfpipty) [reply .editorconfig:4]: good

**Review by alkfpipty — APPROVED** [review]

**Review by melstone — APPROVED** [review]

**Pull request merged**